src/list.js

```diff
@@ -0,0 +1,24 @@
+export default class Task {
+  constructor(taskEntry) {
+    this.taskEntry = taskEntry;
+    this.completed = false;
+    this.index = 0;
+  }
+
+  addTask(taskList) {
+    const myTask = new Task(this.taskEntry);
+    myTask.index = taskList.length;
+    taskList.push(myTask);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+  }
+
+  static delTask(index, taskList) {
+    taskList = taskList.filter((tasks, i) => index !== i);
+    localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    window.location.reload();
+    taskList.forEach((taskindex, i) => {
+      taskindex.index = i;
+      localStorage.setItem('taskDetail', JSON.stringify(taskList));
+    });
+  }
```

Comment:
- [ ] Good job implementing functions to add and delete todos, kindly add a function to implement the editing functionality. Make sure that after editing, the todo task is saved, and its persistent on reload.